packages/terra-tabs/src/common-tabs/_Tabs.module.scss

```diff
@@ -21,4 +22,13 @@
       visibility: hidden;
     }
   }
+
+  .container {
+    display: -webkit-box; // stylelint-disable value-no-vendor-prefix
```

Comment:
an short explanation of why this style is needed will help for maintainability

Reply:
This is added because while using the Flex the style of addIcon getting align to all right to the tabs.

Reply:
you can add the similar details as comment on style file as well

Reply:
Added fce10235910d944bfe5aab215063a7faecce8f86